.github/workflows/release-candidate.yml

```diff
@@ -44,7 +44,7 @@ jobs:
           POSTHOG_PROJECT_ID: ${{ secrets.POSTHOG_PROJECT_ID_RC }}
           POSTHOG_PROJECT_API_KEY: ${{ secrets.POSTHOG_PROJECT_API_KEY_RC }}
           SENTRY_DSN: ${{ secrets.SENTRY_DSN_RC }}
-          SENTRY_ENV: ${{ secrets.MITOPEN_ENV_RC }}
+          SENTRY_ENV: ${{ secrets.MITOPEN_ENVIRONMENT_RC }}
```

Comment:
Honestly, for these values since we already have the workflows split by environment, we should probably just hard-code it in the YAML.

Reply:
That was my initial thought, except Django also uses these values. Using an env var avoids "Are django and frontend both on same page about prod vs production, rc vs qa vs release-candidate"